packages/integration-tests/src/reactiveHints.test.ts

```diff
@@ -52,6 +52,14 @@ describe("reactiveHints", () => {
     ]);
   });
 
+  it("can do via polymorphic reference", () => {
+    expect(reverseReactiveHint(Author, { books: { comments: "text" } })).toEqual([
+      { entity: Author, fields: [], path: [] },
+      { entity: Book, fields: ["author"], path: ["author"] },
+      { entity: Comment, fields: ["parent", "text"], path: ["parent@Book", "author"] },
```

Comment:
@stephenh: This allows us to differentiate the type of the polymorphic entity these reactions should go through